bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/gems/CandidateAsterismsTreeTableModel.java

```diff
@@ -95,9 +96,11 @@ public Object getValue(Row row) {
         // (XXX There might be another way to get the parent node)
         private Row _parent;
 
+        private final Option<Long> _when;
+
         // A row corresponding to a GemsGuideStars object
         // (top level node, displays the Strehl info and checkbox)
-        Row(GemsGuideStars gemsGuideStars, List<Row> children) {
+        Row(GemsGuideStars gemsGuideStars, List<Row> children, Option<Long> when) {
```

Comment:
The point in time used by AGS and the one we use when displaying guide-star coordinates are not necessarily the same, but in practice they will be because we're currently using the start of the scheduling block everywhere. Does this answer your question?